eoscostarica/lifebank
# refactor(backend): improve functions performance

Resolves #698 
Resolves #699  
Resolves #700  

### What does this PR do?
Change email onboarding content

### How should this be tested?
- Make run
- Sign up as donor
- Verify account
- Check onboarding email
